### PR TITLE
Deterministic account derivation with seed

### PR DESCRIPTION
**Pull request by piotr-roslaniec**
## Motivation

With these changes, WASM-bindings users can take advantage of RNG bundled with SDK to generate accounts deterministically. 

## Test Plan

Tested by modifying the existing test suite.

## Related PRs

(No related PRs)

### CODE REVIEW

File: wasm/src/account/account.rs [hunk 1]
```diff
@@ -47,6 +47,18 @@ impl Account {
         }
     }
 
+    #[wasm_bindgen]
+    pub fn from_seed(seed: &[u8]) -> Self {
```
Comment (collinc97) [inline wasm/src/account/account.rs:51]: This method can be shortened. In addition, you should use `ChaChaRng`.
```rust
AccountNative::new(&mut ChaChaRng::seed_from_u64(seed))
```
Please see our implementation in the CLI of this repository for an example.

**Review by collinc97 — CHANGES_REQUESTED** [review]
Thanks for the PR!
It is close to what we want but with a few changes.

**Comment by howardwu** [comment]
@piotr-roslaniec checking in here. Is this PR still relevant?

**Comment by piotr-roslaniec** [comment]
@howardwu Yes, I still need a way to generate `Account` (or `PrivateKey`) in a deterministic way.
I will attempt to rebase this PR and adjust changes to `testnet3`.

**Pull request closed without merging**